test/sdk.test.ts

```diff
@@ -456,6 +456,29 @@ describe('Tests the SDK functionality', () => {
       expectNotIntegration('NativeFrames');
     });
 
+    it('no stall tracking integration by default', () => {
+      init({});
+
+      expectNotIntegration('StallTracking');
+    });
+
+    it('when tracing enabled stall tracking integration added by default', () => {
+      init({
+        tracesSampleRate: 0.5,
```

Comment:
Q: If we need tracesSampleRate in order to enable stall tracking, shouldn't we also disable it if traceSamplerate is 0?

Reply:
I thought about it as well, but I think changing the other options based on other options just adds code which effectively does nothing.

The enable options are effectively just shortcuts for adding the integrations.

But if there is a case which would break we should change it.